test/runtests.jl

```diff
@@ -2,7 +2,7 @@ using Tau
 using Test
 
 @testset "self-identity" begin
-    @test tau isa Irrational{:twoπ}
+    @test tau isa Tau.IrrationalConstants.Twoπ
```

Comment:
Double checking: this works with both `IrrationalConstants` 0.1 and 0.2?

Reply:
Well, no, the test doesn't. Of course, the package/constant works with either version but it's a different type since IrrationalConstants changed the types. I guess a better test might be based on `twoπ` directly:
```suggestion
    @test tau === Tau.IrrationalConstants.twoπ
```

Reply:
I changed it, I think it's better now @giordano.

The question made me wonder whether we should be also make a breaking release, as we did in IrrationalConstants, just to be sure that the different type does not break dispatches of some user. It seems less likely though, at least according to Juliahub no packages depend on Tau.

Reply:
I think we have the following two possibilities.

- Do not consider this PR as breaking, and update the compat table with `0.1, 0.2`.
- Consider this PR as breaking, and update the compat table with `0.2`.

The current behaviour in this PR

- Consider this PR as breaking, and update the compat table with `0.1, 0.2`.

is inconsistent because users may experience surprising type changes in the following situation.

- The user depends on Tau.jl v2.0.0.
- The user depends on SomePkg.jl that depends on IrrationalConstants.jl.
- SomePkg.jl updates its compat table of IrrationalConstants.jl from `0.1` to `0.1, 0.2` without a breaking change.
- Then, the type of `Tau.tau` will be changed.

Reply:
Yes, I guess you're right and it's safer to drop support for IrrationalConstants 0.1.